restore-environment/action.yml

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+name: 'restore-environment'
+description: 'Power Platform Restore Action'
+inputs:
+  source-url:
+    description: 'Source URL of Power Platform environment; e.g. "https://test-env-source.crm.dynamics.com"'
+    required: true
+
+  target-url:
+    description: 'Target URL of Power Platform environment; e.g. "https://test-env-target.crm.dynamics.com"'
```

Comment:
nit: should be a different domain/url than source? Or are identical domain/url legal for restore (i.e. does it work)? If it doesn't, add a check in index.ts as well

Reply:
I just tried to test it with identical domain/urls for source and target with pac cli and it seems valid to use same urls.

Reply:
thanks!